app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,7 @@
   </head>
 
   <body class="prose">
+    <%= render 'layouts/navbar' %>
```

Comment:
You can add the navbar in shared folder like:

/views/shared/_navbar.html.erb

And render like: 

<%= render 'shared/navbar' %>

Reply:
done